src/module-commons-ui.d.ts

```diff
@@ -2,3 +2,5 @@
 declare module '@gridsuite/commons-ui' /*{
     export = typeof import('@gridsuite/commons-ui');
```

Comment:
remove comments

Reply:
will do in another pr to lighten this one